### PR TITLE
49-create-tag-route

### PR DESCRIPTION
**Pull request by souljorn**
gh-49

### CODE REVIEW

**Comment by dacabdi** [comment]
@souljorn remember to include the closes #49 directive to close the issues on merge.

**Comment by dacabdi** [comment]
@souljorn fixed the merge conflicts.

File: dotnet/DataEngine/Tests/Tests/TagTests.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        private readonly string desc = "electrician";
+        private int currentId;
+        private readonly string currentRoute = "/tag/"; 
+        
+        public TagTest()
+        {
+            //Get relative path to DataEngine
+            var localDir = Directory.GetCurrentDirectory();
+            var index = localDir.LastIndexOf("DataEngine/", StringComparison.InvariantCulture);
+            localDir = localDir.Remove(index + 11);
```
Comment (dacabdi) [inline dotnet/DataEngine/Tests/Tests/TagTests.cs]: @souljorn if the parent subdir name changes (ie, no longer 11 chars) you would have to refactor the code. Will change to a more graceful mechanism to travel up.

Comment (souljorn) [reply dotnet/DataEngine/Tests/Tests/TagTests.cs]: Fixed

**Comment by souljorn** [comment]
closes #59

**Pull request merged**